render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/blockfactories/MergingStrategy.java

```diff
@@ -0,0 +1,6 @@
+package org.janelia.render.client.newsolver.blockfactories;
+
+public enum MergingStrategy {
+	RANDOM_PICK,
+	LINEAR_BLENDING;
+}
```

Comment:
consider adding a "list of models" supplier and moving the inetrpolateModels and pickRandom logic from the solver to here

Reply:
I'm not sure I understand your comment. The logic in `interpolateModels` is not (yet) generic, but depends heavily on the model used in intensity correction and alignment. There already is a TODO for moving these two methods (and I agree that this enum is the canonical target) as soon as there is a proper model (not `List<...>`) for intensity correction.

Reply:
my mistake - thought logic could move here already - current todos are sufficient